rosetta/services/network.go

```diff
@@ -55,7 +57,16 @@ func (s *NetworkAPI) NetworkStatus(
 	}
 
 	// Fetch relevant headers, syncing status, & peers
-	currentHeader, err := s.hmy.HeaderByNumber(ctx, rpc.LatestBlockNumber)
+	currBlock := s.hmy.CurrentBlock()
+	var currentHeader *block.Header
+	var err error
+	if currBlock.Number().Cmp(big.NewInt(0)) == 1 && !s.hmy.IsStakingEpoch(currBlock.Epoch()) {
```

Comment:
block 0? can you add a bit comment here?

Reply:
The comment directly below it should be clear enough.
```
// all blocks in the era before staking epoch requires the next block to get the block reward transactions
```

The block 0 check is an edge case check so that we don't attempt to look for block -1.